src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the start of the class definition.